htdocs/comm/action/class/cactioncomm.class.php

```diff
@@ -172,7 +172,7 @@ public function liste_array($active = '', $idorcode = 'id', $excludetype = '', $
 		if ($morefilter) {
 			$sql .= " AND ".$morefilter;
 		}
-		$sql .= " ORDER BY type, position, module";
+		$sql .= " ORDER BY position";
```

Comment:
The position was introduced to allow to sort into combo list of types but this combo is grouped by type
![image](https://user-images.githubusercontent.com/883887/228349639-0e563b7b-f145-4f73-b067-0447e0efd3ab.png)

This is why we have a group by type (to avoid to a a type into the wrong family)